packages/components/src/common/CloseButton/CloseButton.tsx

```diff
@@ -31,32 +31,39 @@ type CloseButtonProps = {
 /**
  * Generic close button.
  */
-const CloseButton = ({
-  className,
-  color = "neutral",
-  onClose,
-  size = "2rem",
-  "aria-label": ariaLabel = "close",
-  ...rest
-}: CloseButtonProps) => (
-  <Button
-    className={clsx(
-      styles.button,
+const CloseButton = forwardRef<HTMLButtonElement, CloseButtonProps>(
+  (
+    {
       className,
-      // Color props
-      color === "brand" && styles.colorBrand,
-      color === "neutral" && styles.colorNeutral,
-      color === "success" && styles.colorSuccess,
-      color === "warning" && styles.colorWarning,
-      color === "alert" && styles.colorAlert,
-    )}
-    color={color}
-    onClick={onClose}
-    variant="link"
-    {...rest}
-  >
-    <CloseRoundedIcon purpose="informative" size={size} title={ariaLabel} />
-  </Button>
+      color = "neutral",
+      onClose,
+      size = "2rem",
+      "aria-label": ariaLabel = "close",
+      ...rest
+    },
+    ref,
+  ) => (
+    <Button
+      className={clsx(
+        styles.button,
+        className,
+        // Color props
+        color === "brand" && styles.colorBrand,
+        color === "neutral" && styles.colorNeutral,
+        color === "success" && styles.colorSuccess,
+        color === "warning" && styles.colorWarning,
+        color === "alert" && styles.colorAlert,
+      )}
+      color={color}
+      onClick={onClose}
+      variant="link"
+      ref={ref}
+      {...rest}
+    >
+      <CloseRoundedIcon purpose="informative" size={size} title={ariaLabel} />
+    </Button>
+  ),
 );
+CloseButton.displayName = "CloseButton";
```

Comment:
We have to manually add a `displayName` when adding `forwardRef`. 🤷

Reply:
Is that a lint rule making us do this? 🤔 

An alternative may be to define the forward ref like:

```js
const CloseButton = forwardRef<HTMLButtonElement, Props>(function CloseButton(props, ref) {
  // ...
});
```

Basically give the function a name that React can pull directly from.

Doesn't really matter either way. Both require duplicating the CloseButton name. Thought just occured to me (and https://reactjs.org/docs/forwarding-refs.html#displaying-a-custom-name-in-devtools indicates it would work)